src/UserCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+
+import { User } from "./ourtypes";
+import { Role } from "./List";
+import UserDisplay from "./UserDisplay";
+import FriendStatusButton from "./FriendManagers";
+
+interface Extras {}
```

Comment:
Don't know if this is necessary if you're not injecting extra properties, but if my shoddy typescripting is requiring it my apologies, and ignore this comment.